.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, actuated-arm64-6cpu-8gb]
+        os: [ubuntu-20.04, ubuntu-24.04, actuated-arm64-6cpu-8gb]
```

Comment:
The comment here is still `for Ubuntu 22.04`.
https://github.com/opencontainers/runc/blob/3fb6a75be988620651298ea7b81d77b34fca653a/.github/workflows/test.yml#L183

Reply:
Let's try to remove it :)

Reply:
Nope, we have to keep it. Changed the text to "22.04 and later".